src/app.ts

```diff
@@ -837,6 +837,17 @@ const awsx = (): void => {
         }
       }
     })
+    .command({
+      command: 'status',
+      describe: 'Show the status of your current awsx session',
+      handler: (): void => {
+        try {
+          console.log(chalk.green(`Current profile is '${currentProfile}'`));
```

Comment:
This is never going to throw. I think for a proper status command we should try to make some kind of "ping" request to AWS and then indicate which profile is active and if it's able to reach and authenticate with AWS.